bitnami/mariadb-galera/values.yaml

```diff
@@ -366,6 +366,8 @@ persistence:
 #   image: bitnami/minideb:stretch
 #   command: ['do', 'something']
 
+## Additional containers
+extraContainers: |
```

Comment:
Do the same in the `values-production.yaml`

Reply:
This is consistent with `extraInitContainers`.

Reply:
Well, sorry I wanted to write: `extraContainers: []` as it could be an array I think it is more concrete. It would be great if you could change both.
That is the approach used for [nginx-ingress-controller](https://github.com/bitnami/charts/blob/f4e342c0a5f83302216fa8d188acbfad2b7114df/bitnami/nginx-ingress-controller/values-production.yaml#L324) too.

Reply:
Done.